rclpy/rclpy/__init__.py

```diff
@@ -32,6 +33,8 @@ def init(*, args=None):
 def create_node(node_name, *, namespace=None):
     namespace = namespace or ''
     failed = False
+    if not ok():
```

Comment:
Since `ok()` is being used now the `# noqa` comment on the import line is obsolete.